File: C1.UWP.FlexChart/CS/SunburstIntro/Converter.cs
using C1.Chart;
using System;
using Windows.UI.Xaml.Data;

namespace SunburstIntro
{
    public class EnumToStringConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            return value.ToString();
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            if ( (parameter as string) == "PieLabelPosition")
                targetType = typeof(PieLabelPosition);
            else if ((parameter as string) == "PieLabelOverlapping")
                targetType = typeof(PieLabelOverlapping);
            return Enum.Parse(targetType, value.ToString());
        }
    }

    public class StringToEnumConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            if (targetType == typeof(Position))
            {
                return (Position)Enum.Parse(typeof(Position), value.ToString());
            }

            return null;
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            throw new NotImplementedException();
        }
    }
}

File: C1.UWP.FlexChart/VB/SunburstIntro/Converter.vb
Imports C1.Chart

Public Class EnumToStringConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return value.ToString()
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Return [Enum].Parse(targetType, value.ToString())
    End Function
End Class

Public Class StringToEnumConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        If targetType Is GetType(Position) Then
            Return CType([Enum].Parse(GetType(Position), value.ToString()), Position)
        End If

        Return Nothing
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Throw New NotImplementedException()
    End Function
End Class